packages/parrot-middleware/README.md

```diff
@@ -2,9 +2,12 @@
 
 parrot-middleware is an implementation of Parrot that can be used as an [Express](http://expressjs.com/) middleware.
 
+Given a collection of Parrot scenarios which each have a list of request and response pairs, the parrot middleware will intercept the `request`s that match the paths in the active scenario, returning the corresponding `response`, in addition to providing the three additional endpoints detailed below for interacting with your scenarios.
```

Comment:
I think the last bit of the sentence (starting with `in addition to...` is confusing, I am not sure what it is trying to say.

Reply:
Revised this, let me know if you think this is better...